opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -535,6 +536,7 @@ def to_json(self, indent=4):
         f_span["context"] = self._format_context(self.context)
         f_span["kind"] = str(self.kind)
         f_span["parent_id"] = parent_id
+        f_span["is_recording"] = self.is_recording()
```

Comment:
Upon span ending, console exporter would receive a span but would print out `False` for `is_recording` because span has already ended. Wouldn't this be confusing?

Reply:
Hm, I think it would be clear since the `end_time` and `status` would both be set as well and the overall intent of this change is to reflect an `is_recording` value of false when the span is ended. Not tied to this though, so happy to remove if desired.